lib/middleware.js

```diff
@@ -28,7 +37,7 @@ function extend(context, options) {
     }, options.serializers)
   });
 
-  return _.assign(newContext, { log, child });
+  return _.assign(newContext, { log, child, refProps });
```

Comment:
`refProps` should be omitted in the child serializer. Btw, since this is an internal property, maybe we should have it prefixed with `_` to avoid collisions?

Reply:
is it not possible though to call child on a child? would we not want refProps there to avoid the same deep copy issue?

Reply:
omitting property in a serializer just prevents it from showing up in logs

Reply:
aah sorry, didn't see the context serializer